crates/rattler/src/install/driver.rs

```diff
@@ -130,6 +140,31 @@ impl InstallDriver {
             );
         }
     }
+
+    /// Return a locked reference to the paths registry. This is used to make sure that the same
+    /// path is not installed twice.
+    pub fn clobber_registry(&self) -> MutexGuard<ClobberRegistry> {
+        self.clobber_registry.lock().unwrap()
+    }
+
+    /// Call this after all packages have been installed to perform any post processing that is
+    /// required.
+    ///
+    /// This function will select a winner among multiple packages that might write to a single package
+    /// and will also execute any `post-link.sh/bat` scripts
+    pub fn post_process(
```

Comment:
Would it make sense to give this a more descriptive name, like `unclobber` or something similar?

Reply:
idk. post_process is more generic so we could potentially do some other things as well in this step ...